### ISSUE TITLE
UI display scripts do not work correctly on a game reload.

### ISSUE THREAD

**Issue by Scrivener07** [Scrivener07/FO4_Games#51]
OnInit only occurs once ever during the entire lifetime of a script. The problem is likely the function `OnInit` on the [Games:Shared:UI:Display](https://github.com/Scrivener07/FO4_Games/blob/66bf0165b7923e4622152f7e9cb695a9398ae07e/Data/Scripts/Source/FO4_Games/Games/Shared/UI/Display.psc#L13-L17). Maybe F4SE/Scaleform does not persist injected UI elements across game reloads? I will try to use the [OnPlayerLoadGame - Actor](https://www.creationkit.com/fallout4/index.php?title=OnPlayerLoadGame_-_Actor) event to reload the UI display onto the target menu.

**Papyrus**
```
Event OnInit()
	If (Data())
		Load()
	EndIf
EndEvent
```

**Comment by Scrivener07** [comment]
Instead of loading only inside of `OnInit`, I now also do a reload with the `OnPlayerLoadGame` event.

[ Games:Shared:UI:Display](https://github.com/Scrivener07/FO4_Games/blob/256e24f778255734d1bfb6e25ae3f0d73d206a5d/Data/Scripts/Source/FO4_Games/Games/Shared/UI/Display.psc#L13-L30)
```
Event OnInit()
	RegisterForRemoteEvent(Game.GetPlayer(), "OnPlayerLoadGame")
	OnGameReload()
	WriteLine(self, "Initialization is complete for "+ToString())
EndEvent


Event Actor.OnPlayerLoadGame(Actor akSender)
	OnGameReload()
	WriteLine(self, "Reload is complete for "+ToString())
EndEvent


Event OnGameReload()
	If (Data())
		Load()
	EndIf
EndEvent
```

**Comment by Scrivener07** [comment]
Fixed with commit 256e24f
